src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -108,7 +110,41 @@ public SearchReadsRequest getReadsRequest(String readGroupSetId) {
         .setEnd(end);
   }
 
-  public static Iterable<Contig> parseContigsFromCommandLine(String contigsArgument) {
+  public StreamReadsRequest getStreamReadsRequest(String readGroupSetId) {
```

Comment:
Same thing for those, though I understand if you want to do that in a separate PR.